bundle/manifests/sonataflow-operator.clusterserviceversion.yaml

```diff
@@ -101,7 +101,7 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/kiegroup/kogito-serverless-operator
     support: Red Hat
-  name: sonataflow-operator.v1.41
+  name: sonataflow-operator.v1.41.0-snapshot
```

Comment:
We need to figure a way to remove this, since OLM won't accept this suffix @radtriste